src/Glpi/Application/SystemConfigurator.php

```diff
@@ -248,6 +248,7 @@ private function setSessionConfiguration(): void
         // The cookie name contains the root dir + HTTP host + HTTP port to ensure that it is unique
         // for every GLPI instance, enven if they are served by the same server (mostly for dev envs).
         session_name('glpi_' . \hash('sha512', $this->root_dir . ($_SERVER['HTTP_HOST'] ?? '') . ($_SERVER['SERVER_PORT'] ?? '')));
+        $_SESSION['glpi_currenttime'] = date("Y-m-d H:i:s");
```

Comment:
Won't this create issues with timezone ? 
The only time we set this value is in the `setTimezone()` function.

Reply:
This is set in src/DBmysql.php, src/Glpi/Application/SystemConfigurator.php and src/Session.php.
This variable is meant to be used to get rid of (micro)seconds creation time of all database entries. I guess settign a known value is less problametic than having a missing one (and this currently should only happen from e2e tests suite - in phpunit we explicitely set it when needed already).

Reply:
There are lots of warnings in logs during e2e tests. I'm not going to spend more time on that, commit is reverted.